src/sensor/inc/Sensor.h

```diff
@@ -13,16 +13,15 @@
 #include <string>
 #include <arpa/inet.h>
 
-#include "../../core/inc/Serializer.h"
-#include "../../core/inc/Connection.h"
 #include "SensorConfig.h"
 #include <vector>
+#include <UdpConnection.h>
 
 class Sensor {
 private:
     Serializer serializer;
-    Connection* con_send;
-    Connection* con_recv;
+    UdpConnection con_send;
```

Comment:
Nie referencje a zwykłe pola. Są one alokowane wtedy razem z resztą sensora. A to nie boli bo UdpConnection to boolean i integer. Leciutka klasa a niealokowanie pamięci ręcznie upraszcza kod.